### PR TITLE
docs: Rancher UI on Rancher Desktop

### PR DESCRIPTION
**Pull request by evertonlperes**
### Changes
1. Added a new Tutorial item: Rancher UI on Rancher Desktop.

### CODE REVIEW

**Comment by evertonlperes** [comment]
@gunamata 
I'm opening this draft with a new section called `Examples`. This section has the objective to provide some use cases to the users for local development.

**Comment by gunamata** [comment]
yes, these examples are helpful.. we need to decide on the section title though, whether to call it `Tutorial` or `Examples`.. 

for example, I have another  #6  open where I wrote a doc on how to use VS Code remote containers with RD.. I used the `Tutorial` as the section title though.

**Comment by evertonlperes** [comment]
> yes, these examples are helpful.. we need to decide on the section title though, whether to call it `Tutorial` or `Examples`..
> 
> for example, I have another #6 open where I wrote a doc on how to use VS Code remote containers with RD.. I used the `Tutorial` as the section title though.

Oh I see, cool.
I'll let your tutorial get merged first and I can modify mine with the correct title and such. Thanks for the heads up.

Thanks @jtravee for adding the labels, I missed to include those 🤣

File: docs/tutorials/rancher-ui-on-rancher-desktop.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+title: Rancher UI on Rancher Desktop
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Installing Rancher Dashboard on Rancher Desktop using `container runtime` or `helm` (local environment):
```
Comment (jtravee) [inline docs/tutorials/rancher-ui-on-rancher-desktop.md:8]:
```suggestion
To install Rancher Dashboard on Rancher Desktop using `container runtime` or `helm` (local environment):
```

File: docs/tutorials/rancher-ui-on-rancher-desktop.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  <TabItem value="docker" default>
+
+```console
+docker run --privileged -d --restart=no -p 8080:80 -p 8443:443 rancher/rancher
+```
+
+  </TabItem>
+  <TabItem value="helm" default>
+
+Step 1: Add jetstack charts
```
Comment (jtravee) [inline docs/tutorials/rancher-ui-on-rancher-desktop.md:27]:
```suggestion
1: Add Jetstack charts:
```

File: docs/tutorials/rancher-ui-on-rancher-desktop.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+```console
+helm repo add jetstack https://charts.jetstack.io
+```
+
+Step 2: Add rancher lastest charts
```
Comment (jtravee) [inline docs/tutorials/rancher-ui-on-rancher-desktop.md:32]:
```suggestion
2: Add latest Rancher charts:
```

File: docs/tutorials/rancher-ui-on-rancher-desktop.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+```console
+helm repo add rancher-latest https://releases.rancher.com/server-charts/latest
+```
+
+Step 3: Create cert-manager namespace
```
Comment (jtravee) [inline docs/tutorials/rancher-ui-on-rancher-desktop.md:37]:
```suggestion
3: Create cert-manager namespace:
```

File: docs/tutorials/rancher-ui-on-rancher-desktop.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+```console
+kubectl create namespace cert-manager
+```
+
+Step 4: Install cert-manager services
```
Comment (jtravee) [inline docs/tutorials/rancher-ui-on-rancher-desktop.md:42]:
```suggestion
4: Install cert-manager services:
```

File: docs/tutorials/rancher-ui-on-rancher-desktop.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+```console
+helm install cert-manager jetstack/cert-manager --namespace cert-manager --version v1.1.0 --set installCRDs=true
+```
+
+Step 5: Create cattle-system namespace
```
Comment (jtravee) [inline docs/tutorials/rancher-ui-on-rancher-desktop.md:47]:
```suggestion
5: Create cattle-system namespace:
```

File: docs/tutorials/rancher-ui-on-rancher-desktop.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+```console
+kubectl create namespace cattle-system
+```
+
+Step 6: Install Rancher application
```
Comment (jtravee) [inline docs/tutorials/rancher-ui-on-rancher-desktop.md:52]:
```suggestion
6: Install Rancher application:
```

File: docs/tutorials/rancher-ui-on-rancher-desktop.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+```console
+helm install rancher rancher-latest/rancher --namespace cattle-system --set hostname=rancher.rd.localhost --wait --timeout=10m
+```
+
+  </TabItem>
+</Tabs>
+
+The installation takes a few minutes to complete. After the installation, you can access Rancher UI:
```
Comment (jtravee) [inline docs/tutorials/rancher-ui-on-rancher-desktop.md:60]:
```suggestion
The installation takes a few minutes to complete. After the installation, you can access the Rancher UI as follows: 
```

File: docs/tutorials/rancher-ui-on-rancher-desktop.md [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+
+![](/img/examples/rancherUiWelcomePage.png)
+
+
+<Tabs groupId="container-runtime">
+  <TabItem value="nerdctl" default>
+
+In order to access the Rancher UI, you need to get the Bootstrap Password:
+
+Step 1: Get Rancher UI container ID/Name
```
Comment (jtravee) [inline docs/tutorials/rancher-ui-on-rancher-desktop.md:72]:
```suggestion
1: Get Rancher UI container ID/Name:
```

File: docs/tutorials/rancher-ui-on-rancher-desktop.md [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+```console
+nerdctl ps
+```
+Step 2: Get the Bootstrap Password
```
Comment (jtravee) [inline docs/tutorials/rancher-ui-on-rancher-desktop.md:76]:
```suggestion
2: Get the Bootstrap Password:
```

File: docs/tutorials/rancher-ui-on-rancher-desktop.md [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+```console
+nerdctl logs [rancherContainerID] 2>&1 | grep "Bootstrap Password:"
+```
+Step 3: Bootstrap Password example
```
Comment (jtravee) [inline docs/tutorials/rancher-ui-on-rancher-desktop.md:80]:
```suggestion
3: Bootstrap Password example:
```

File: docs/tutorials/rancher-ui-on-rancher-desktop.md [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+```console
+[INFO] Bootstrap Password: 7fwjjw4ldcmnq8ghns22q7nhl5lrznwwt9p9vjljfjc6tqbcvhxmwq
+```
+
+  </TabItem>
+  <TabItem value="docker" default>
+
+In order to access the Rancher UI, you need to get the Bootstrap Password:
+
+Step 1: Get Rancher UI container ID/Name
```
Comment (jtravee) [inline docs/tutorials/rancher-ui-on-rancher-desktop.md:90]:
```suggestion
1: Get Rancher UI container ID/Name:
```

File: docs/tutorials/rancher-ui-on-rancher-desktop.md [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+```console
+docker ps
+```
+Step 2: Get the Bootstrap Password
```
Comment (jtravee) [inline docs/tutorials/rancher-ui-on-rancher-desktop.md:94]:
```suggestion
2: Get the Bootstrap Password:
```

File: docs/tutorials/rancher-ui-on-rancher-desktop.md [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+```console
+docker logs [rancherContainerID] 2>&1 | grep "Bootstrap Password:"
+```
+Step 3: Bootstrap Password example
```
Comment (jtravee) [inline docs/tutorials/rancher-ui-on-rancher-desktop.md:98]:
```suggestion
3: Bootstrap Password example:
```

File: docs/tutorials/rancher-ui-on-rancher-desktop.md [hunk 15]
```diff
@@ -0,0 +1,109 @@
...
+```console
+[INFO] Bootstrap Password: 7fwjjw4ldcmnq8ghns22q7nhl5lrznwwt9p9vjljfjc6tqbcvhxmwq
+```
+  </TabItem>
+</Tabs>
+
+Follow the wizard instructions and click on `Continue`, to land on the main Rancher UI page.
```
Comment (jtravee) [inline docs/tutorials/rancher-ui-on-rancher-desktop.md:105]:
```suggestion
Follow the wizard instructions and click on `Continue` to land on the main Rancher UI page.
```

File: docs/tutorials/rancher-ui-on-rancher-desktop.md [hunk 16]
```diff
@@ -0,0 +1,109 @@
...
+
+![](/img/examples/rancherUiMainPage.png)
+
+From the Rancher UI you can manage your local cluster, node and more. For more information, see [Rancher Docs](https://rancher.com/docs/).
```
Comment (jtravee) [inline docs/tutorials/rancher-ui-on-rancher-desktop.md:109]:
```suggestion
From the Rancher UI, you can manage your local cluster, node, and more. For more information, see [Rancher Docs](https://rancher.com/docs/).
```

**Review by jtravee — CHANGES_REQUESTED** [review]

File: docs/tutorials/rancher-ui-on-rancher-desktop.md [hunk 17]
```diff
@@ -0,0 +1,109 @@
...
+
+  </TabItem>
+  <TabItem value="helm" default>
+
+1: Add Jetstack charts:
+```console
+helm repo add jetstack https://charts.jetstack.io
+```
+
+2: Add lastest Rancher charts:
```
[the remark this answers is not in the document]
Comment (evertonlperes) [reply docs/tutorials/rancher-ui-on-rancher-desktop.md]: Oh boy.. my bad!
Thanks for catching it :D

**Review by gunamata — APPROVED** [review]

**Pull request merged**